src/main/scala/org/clulab/habitus/actions/HabitusActions.scala

```diff
@@ -170,8 +172,27 @@ class HabitusActions extends Actions {
       )
     }
   }
+
+  def yieldAmountActionFlow(mentions: Seq[Mention]): Seq[Mention] = {
+//    splitIntoBinary(mentions).filter(m => allowableTokenDistanceBetweenVarAndValue(m, 16))
+    appropriateMeasurement(splitIntoBinary(mentions).filter(m => allowableTokenDistanceBetweenVarAndValue(m, 16)))
+  }
+
+  def fertilizerQuantityActionFlow(mentions: Seq[Mention]): Seq[Mention] = {
+//    splitIntoBinary(mentions).filter(m => allowableTokenDistanceBetweenVarAndValue(m, 12))
+    appropriateMeasurement(splitIntoBinary(mentions).filter(m => allowableTokenDistanceBetweenVarAndValue(m, 12)))
+  }
+
+  def allowableTokenDistanceBetweenVarAndValue(mention: Mention, maxDist: Int): Boolean = {
+    val variable = mention.arguments("variable").head
+    val value = mention.arguments("value").head
```

Comment:
I count 14 uses of "value" and 8 of "variable".  I think that's sufficient to turn the string into a variable and reference it.  The compiler can check that somebody (me) has not misspelled it.  Something like
```scala
val VALUE = "value"
mention.arguments(VALUE).head
```

Reply:
Okay. That’s good.

Reply:
There may be a few more instances than what you found so far.